### PR TITLE
Add improved status icon support on the task list

### PR DESCRIPTION
**Pull request by TwistMeister**
### Changes
Added different icons and styles for statusses of members and tasks lists

### Issue link
Closes https://github.com/minvws/nl-kat-coordination/issues/1168 

### Proof
image

image

---

### Code Checklist
- [x] All the commits in this PR are properly PGP-signed and verified;
- [x] This PR only contains functionality relevant to the issue; tickets have been created for newly discovered issues.
- [ ] I have written unit tests for the changes or fixes I made.
- [ ] For any non-trivial functionality, I have added integration and/or end-to-end tests.
- [x] I have performed a self-review of my code and refactored it to the best of my abilities.

---
## Checklist for code reviewers:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---
## Checklist for QA:
Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.

### CODE REVIEW

**Comment by Darwinkel** [comment]
## Checklist for QA:
- [x] I have checked out this branch, and successfully ran a fresh `make reset`.
- [x] I confirmed that there are no unintended functional regressions in this branch:
    - [x] I have managed to pass the onboarding flow
    - [x] Objects and Findings are created properly
    - [x] Tasks are created and completed properly
- [x] I confirmed that the PR's advertised `feature` or `hotfix` works as intended.

### What works:
* New icons work on the members page, boefjes task page, and normalizer task page for all implemented states

**Review by ammar92 — APPROVED** [review]

**Review by ammar92 — APPROVED** [review]

**Review by ammar92 — DISMISSED** [review]

File: rocky/rocky/locale/nl/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -4446,6 +4446,12 @@ msgstr "Super gebruiker"
 msgid "Active"
 msgstr "Actief"
 
+#: rocky/templates/organizations/organization_member_list.html
+#, fuzzy
+#| msgid "new"
+msgid "New"
+msgstr "nieuw"
```
Comment (dekkers) [inline rocky/rocky/locale/nl/LC_MESSAGES/django.po:4453]: The fuzzy matching is incorrectly missing capitalization.

File: rocky/rocky/locale/pap/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -4437,6 +4437,12 @@ msgstr "Super usario"
 msgid "Active"
 msgstr "Aktivá"
 
+#: rocky/templates/organizations/organization_member_list.html
+#, fuzzy
+#| msgid "new"
+msgid "New"
+msgstr "nobo"
```
Comment (dekkers) [inline rocky/rocky/locale/pap/LC_MESSAGES/django.po:4444]: It's missing here too. I also don't know if this is correct. @rieven ?

File: rocky/rocky/templates/tasks/boefjes.html [hunk 3]
```diff
@@ -37,17 +37,21 @@ <h1>{% translate "Boefjes" %}</h1>
...
+                                                <i class="icon running"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "cancelled" %}
+                                                <i class="icon cancelled"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "failed" %}
+                                                <i class="icon negative"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% elif task.status.value == "completed" %}
+                                                <i class="icon positive"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% else %}
+                                                <i class="icon neutral"></i>&nbsp;{{ task.status.value|capfirst }}
+                                            {% endif %}
```
Comment (dekkers) [inline rocky/rocky/templates/tasks/boefjes.html:53]: Not really blocking, but there seems to be a lot of duplicated between these templates that we should maybe refactor in another ticket?

Comment (TwistMeister) [reply rocky/rocky/templates/tasks/boefjes.html:53]: I think you're right. Could you create a ticket with the changes you want to see happening?

**Review by ammar92 — DISMISSED** [review]
Please run `stylelint` once so we can merge this soon

**Review by ammar92 — APPROVED** [review]

**Review by underdarknl — APPROVED** [review]

**Pull request merged**